eggplant/profiles/views.py

```diff
@@ -91,6 +92,7 @@ class Profile(LoginRequiredMixin, FormView):
 
     def get_object(self, queryset=None):
         self.objects = UserProfile.objects.get(user_id=self.request.user.id)
```

Comment:
Regardless of CBV patterns and the parent class' contract, this would be a misleading typo.